core/src/main/java/rs/kunpero/vacation/service/VacationService.java

```diff
@@ -193,7 +197,7 @@ private boolean checkVacationsForUser(String userId, String teamId, LocalDate fr
         return vacationInfoList.stream()
                 .sorted(Comparator.comparing(VacationInfo::getDateFrom))
                 .map(v -> new VacationInfoDto()
-                        .setVacationInfo(String.format("<@%s> `%s` - `%s` %s %s", v.getUserId(), v.getDateFrom(), v.getDateTo(),
+                        .setVacationInfo(String.format("<@%s> `%s` - `%s` %s %s", v.getUserId(), dateTimeFormatter.format(v.getDateFrom()), dateTimeFormatter.format(v.getDateTo()),
```

Comment:
Add info about this property to readme file plz

Reply:
Done